modules/payment/src/Plugin/Commerce/PaymentGateway/PaymentGatewayBase.php

```diff
@@ -107,6 +107,9 @@ protected function getDefaultForms() {
     if ($this instanceof SupportsRefundsInterface) {
       $default_forms['refund-payment'] = 'Drupal\commerce_payment\PluginForm\PaymentRefundForm';
     }
+    if ($this instanceof OffsitePaymentGatewayInterface) {
+      $default_forms['offsite-payment'] = 'Drupal\commerce_payment\PluginForm\OffsitePaymentForm';
```

Comment:
I just realized this is an abstract class. @bojanz we should just remove this, yeah? I think if its missing it'll throw an exception or something.